awx/ui_next/public/index.html

```diff
@@ -1,6 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <% if (process.env.NODE_ENV === 'production') { %>
```

Comment:
@jakemcdermott how does this get set?  Will this take effect for official AWX builds?

Reply:
Yes

Reply:
> how does this get set?

This isn't the final `index.html` that gets packaged in the build or collected by django. It's actually an `.ejs` template. 

Were it configurable by `react-scripts` (it isn't), I would name this file `index.html.j2.ejs`. `react-scripts` processes it as an `.ejs` template to produce the `jinja` template that django uses to serve the actual html file.

Reply:
> Will this take effect for official AWX builds?

Yes

> Will this take effect for official AWX builds?

This isn't the final index.html that gets packaged in the build or collected by django. It's actually an .ejs template.

Were it configurable by react-scripts (it isn't), I would name this file index.html.j2.ejs. react-scripts processes it as an .ejs template to produce the jinja template that django uses to serve the actual html file.

Reply:
Good explanation @jakemcdermott